group_project_v2/public/js/components/submission.js

```diff
@@ -37,7 +37,7 @@ function GroupProjectSubmissionBlock(runtime, element) {
                     technical_details += GroupProjectCommon.gettext(' Technical details: 403 error.');
                 }
 
-                message.content = "<p>" + base_message + "</p><p>" + technical_details + "</p>";
+                message.content = base_message + technical_details;
```

Comment:
@kaizoku Is there a chance to preserve separation between message and technical details? I.e. add them as separate attributes to `message` object and than in function rendering the message put them into different elements, or append a paragraph if `technical_details` is present.

Reply:
This is the other possible solution I was mentioning on the JIRA ticket. We could add a raw parameter to show_message(), and not escape if true for instance, but this may weaken our defenses as if we aren't very careful with this code it could be tricked into not escaping output, and allow us to be vulnerable to XSS still.

If we split the error messages and standard messages, will also have to add extra parameters to show_message for the event handlers, and rendering code, which seems also seems a bit overly complicated.

If this formatting is this important, I can try one or both of these methods.